mace/tools/arg_parser.py

```diff
@@ -543,6 +544,12 @@ def build_preprocess_arg_parser() -> argparse.ArgumentParser:
         default=None,
         required=False,
     )
+    parser.add_argument(
+        "--num_process",
+        help="The user defined number of processes to use", 
```

Comment:
Also add to this comment that this implicitly sets the # of files that will be created for the train, test, and val. Possibly a discussion/warning that if the dataset size is too large/ very small this may need to be tweaked independent of the number of CPUs?

Actually, we may want to consider decoupling the two and having a `num_process` and `num_files` argument?

Reply:
I added the note in the help.